ods_ci/tests/Tests/0600__distributed_workloads/0601__workloads_orchestration/test-pipelines-integration.robot

```diff
@@ -0,0 +1,131 @@
+*** Settings ***
+Documentation       Test suite for OpenShift Pipeline using kfp python package
+
+Resource            ../../../Resources/RHOSi.resource
+Resource            ../../../Resources/ODS.robot
+Resource            ../../../Resources/Common.robot
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Permissions.resource
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource            ../../../Resources/CLI/DataSciencePipelines/DataSciencePipelinesBackend.resource
+Resource            ../../../Resources/Page/DistributedWorkloads/DistributedWorkloads.resource
+Library             DateTime
+Library             ../../../../libs/DataSciencePipelinesAPI.py
+Library             ../../../../libs/DataSciencePipelinesKfp.py
+Test Tags           DataSciencePipelines-Backend
+Suite Setup         Data Science Pipelines Suite Setup
+Suite Teardown      RHOSi Teardown
+
+
+*** Variables ***
+${PROJECT_NAME}=    dw-pipelines
+${KUEUE_RESOURCES_SETUP_FILEPATH}=    tests/Resources/Page/DistributedWorkloads/kueue_resources_setup.sh
+
+
+*** Test Cases ***
+Verify Ods Users Can Create And Run A Data Science Pipeline With Ray Using The kfp Python Package
+    [Documentation]    Creates, runs pipelines with regular user. Double check the pipeline result and clean
+    ...    the pipeline resources.
+    ...    AutomationBugOnDisconnected: RHOAIENG-12514
+    [Tags]      Tier1
+    ...         AutomationBugOnDisconnected
+    ...         DistributedWorkloads
+    ...         WorkloadsOrchestration
+    ...         DataSciencePipelines-DistributedWorkloads
+    Skip If Component Is Not Enabled    ray
+    Skip If Component Is Not Enabled    codeflare
```

Comment:
Those components are enabled by default.
IMHO we should remove these checks as disabled component should fail the test to indicate that something is wrong.